suite-native/transactions/src/utils.ts

```diff
@@ -2,21 +2,37 @@ import { A, G, pipe } from '@mobily/ts-belt';
 
 import { EnhancedVinVout, Target } from '@trezor/blockchain-link-types';
 
+import { VinVoutAddress, AddressesType } from './types';
+
 export const mapTransactionInputsOutputsToAddresses = (
     inputsOutputs: EnhancedVinVout[] | Target[],
```

Comment:
plase make params of this function object so they are "named"

Reply:
fixed in c3086c22c